HomeWork1/HomeWork1.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE HTML>
+<html>
+
+<head>
+    <title>
+        Table
+    </title>
+</head>
+<table align="center" border color="black" style "width:100%">
+    <tr height=2>
+        <th width=200 colspan="6" align="center"><b>TimeTable</b></th>
+    </tr>
+    <tr>
+        <td width=200 rowspan="7" align="center"><b>Hours</b></td>
+    </tr>
+    <tr>
+        <td width=200 align="center"><b>Mon</b></td>
+        <td width=200 align="center"><b>Tue</b></td>
+        <td width=200 align="center"><b>Wed</b></td>
+        <td width=200 align="center"><b>Thu</b></td>
+        <td width=200 align="center"><b>Fri</b></td>
+    </tr>
```

Comment:
Your output is correct.
But, better to put "Hours" and week-days names within one row. Then, you can use rowspan="6" to "Hours". Otherwise, the logic behinds the "colspan and rowspan" is missed in your code.